controllers/authCtrl.js

```diff
@@ -0,0 +1,128 @@
+const Users = require('../models/userModel')
+const bcrypt = require('bcrypt')
+const jwt = require('jsonwebtoken')
+
+const authCtrl = {
```

Comment:
Controll should only have API definibation, please add all business logic inn service file.
Create authService.js, create methods and add all the logic in service file, from contrller just call the service methods